src/Moryx/Workflows/Implementation/Workplan.cs

```diff
@@ -92,5 +94,99 @@ public static Workplan Restore(List<IConnector> connectors, List<IWorkplanStep>
         {
             return new Workplan(connectors, steps);
         }
-    }
+
+        /// <summary>
+        /// Compare two workplans
+        /// </summary>
+        /// <param name="obj"></param>
+        /// <returns></returns>
+        public override bool Equals(object obj)
```

Comment:
Please move this into a static method on `Workplan.Equals(workplanA, workplanB)` instead of overwriting the object method.